Source/SelfService/Web/applications/microservice/microservices/microservicesDataGrid.tsx

```diff
@@ -2,35 +2,37 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import React, { useCallback, useEffect, useState } from 'react';
+
 import { useNavigate } from 'react-router-dom';
 
 import { getPodStatus, MicroserviceObject } from '../../../apis/solutions/api';
 import { HttpResponseApplication } from '../../../apis/solutions/application';
 
 import { DataGridPro } from '@mui/x-data-grid-pro';
-import { Paper } from '@mui/material';
 
-import { microservicesTableColumns } from './microservicesTableColumns';
+import { DataGridWrapper, dataGridDefaultProps } from '@dolittle/design-system';
+
+import { microservicesDataGridColumns } from './microservicesDataGridColumns';
 
-export type MicroserviceTableProps = {
+export type MicroservicesDataGridProps = {
     environment: string;
     application: HttpResponseApplication;
     microservices: MicroserviceObject[];
 };
 
-export const MicroserviceTable = ({ application, environment, microservices }: MicroserviceTableProps) => {
+export const MicroservicesDataGrid = ({ application, environment, microservices }: MicroservicesDataGridProps) => {
     const navigate = useNavigate();
 
     const [microserviceRows, setMicroserviceRows] = useState<(MicroserviceObject | undefined)[]>([]);
```

Comment:
is this the correct type? can the array containe undefined and MicroserviceObject entries, or is it meant to be `MicroserviceObject[] | undefined`?

Reply:
I really do not know. It would take too much time to delve into it. I don't think it's worth it... I know it gives errors that way.